src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +22,49 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { form } = this.props;
+    const { response } = form.submission;
+    const time = _.get('timestamp', response);
+    const name = form.data.veteranFullName;
+    const healthEligibilityCenter = [
+      'Health Eligibility Center',
+      '2957 Clairmont Rd., Suite 200',
+      'Atlanta, GA 30329'
+    ];
 
     return (
       <div>
-        <div className="usa-width-one-sixth medium-2 columns">
-          <i className="fa fa-check-circle hca-success-icon"></i>
-        </div>
-        <div className="usa-width-five-sixths medium-10 columns">
-          <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>Normally processed within <strong>a week</strong>.</p>
+        <p>
+          We may contact you for more information or documents.<br/>
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>Health Care Benefit Claim <span className="additional">(Form 10-10EZ)</span></h4>
+          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+
+          <ul className="claim-list">
+            <li>
+              <strong>Date received</strong><br/>
+              <span>{moment(time).format('MMM D, YYYY')}</span>
+            </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.formSubmissionId}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong>
+              <address className="schemaform-address-view">{healthEligibilityCenter.map((line, index) =>
+                <p className="hca-confirmation-address" key={index}>{line}</p>)}</address>
+            </li>
+          </ul>
         </div>
-        <div>
-          <p>
-            We are processing your application.<br/>
-            The Department of Veterans Affairs will contact you when we finish our review.<br/>
-            <i>Please print this page for your records.</i>
-          </p>
-          {time &&
```

Comment:
@jbalboni is it still necessary to check whether the response includes all of the necessary information? It doesn't look like that check is performed in other confirmation pages (except for some benefit-specific conditional logic as in the 1990 and 5490).

Reply:
Yes, for some reason this confirmation is refreshed a lot and not doing those checks results in Sentry errors.

Reply:
Thanks @jbalboni, changed.

Reply:
Also, would it make sense to use sessionStorage to retain these values across refreshes?

Reply:
If we want people to see the status of forms after submitting, I'd rather come up with a way to do that more securely than putting info in sessionStorage.

Reply:
Thanks @jbalboni, good point, would a message similar to that for SIP (Are you sure you want to navigate away from this page? Doing so will result in ...) assuming one is not already in place?